src/community/jdbcstore/src/main/resources/jdbcstore.properties

```diff
@@ -44,4 +44,4 @@ pool.validationQuery=SELECT now()
 deleteDestinationOnRename=true
 
 # leave the following directories on the filesystem -- no spaces!
-ignoreDirs=data,jdbcstore,workspaces,layergroups,temp,tmp
+ignoreDirs=data,jdbcstore,workspaces,layergroups,temp,tmp,logs
```

Comment:
Going through this list I have some questions:

- data - this is a convention only - for a folder that contains spatial data
- workspaces - not appropriate to ignore workspaces completely, as each workspace styles folder contains styles and icons to be managed.
- layergroups - what resource or config file is stored in the layergroups folder? I am not aware of any ...